Assets/Scripts/Inventory/InventoryManager.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class InventoryManager : MonoBehaviour
+{
+
+    public int potionsAmount = 0;
+
+    [SerializeField]
+    private FirebaseInventoryManager firebase;
+
+    public UnityEvent<int> PotionsUpdated;
+
+    private void Start()
+    {
+        firebase.LoadData();
+    }
+
+    public void UpdatePotionsCount()
+    {
+        if (PotionsUpdated != null) PotionsUpdated.Invoke(potionsAmount);
+    }
```

Comment:
No other class should have to invoke this. The InventoryManager itself should know best.